lib/ansible/modules/cloud/azure/azure_rm_deployment.py

```diff
@@ -543,18 +549,19 @@ def destroy_resource_group(self):
                 self.fail("Delete resource group and deploy failed with status code: %s and message: %s" %
                           (e.status_code, e.message))
 
-    def resource_group_exists(self, resource_group):
+    def _get_resource_group(self, resource_group):
```

Comment:
get_resource_group is available from the base class, so there's no need to have local function here.